core/src/main/scala/kafka/network/RequestConvertToJson.scala

```diff
@@ -63,6 +63,7 @@ object RequestConvertToJson {
       case req: EnvelopeRequest => EnvelopeRequestDataJsonConverter.write(req.data, request.version)
       case req: ExpireDelegationTokenRequest => ExpireDelegationTokenRequestDataJsonConverter.write(req.data, request.version)
       case req: FetchRequest => FetchRequestDataJsonConverter.write(req.data, request.version)
+      case req: GetTelemetrySubscriptionsRequest => GetTelemetrySubscriptionsRequestDataJsonConverter.write(req.data, request.version)
```

Comment:
Not quite in the right place for alphabetical order.

Reply:
Thanks, missed it. My bad.